fbchat/client.py

```diff
@@ -88,11 +88,6 @@ def __init__(self, email, password, debug=True, user_agent=None):
     def _console(self, msg):
         if self.debug: print(msg)
 
-    def _setttstamp(self):
```

Comment:
I'm not a big fan of this. A function is still a function. Things like this might be needed to be called later on.